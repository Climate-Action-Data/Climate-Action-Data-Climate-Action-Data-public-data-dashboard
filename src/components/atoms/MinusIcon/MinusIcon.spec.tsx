import { render } from '@testing-library/react'
import { MinusIcon } from './MinusIcon'

it(`renders correctly`, () => {
  const { container } = render(<MinusIcon />)
  expect(container).toMatchSnapshot()
})
