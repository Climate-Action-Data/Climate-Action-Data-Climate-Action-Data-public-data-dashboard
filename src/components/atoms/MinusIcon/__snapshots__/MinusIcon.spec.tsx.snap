// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <svg
    class="chakra-icon css-n059si"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M5.99997 12.75C5.78747 12.75 5.60936 12.678 5.46563 12.5342C5.32188 12.3904 5.25 12.2122 5.25 11.9997C5.25 11.7871 5.32188 11.609 5.46563 11.4654C5.60936 11.3218 5.78747 11.25 5.99997 11.25H18C18.2125 11.25 18.3906 11.3219 18.5344 11.4657C18.6781 11.6095 18.75 11.7877 18.75 12.0003C18.75 12.2129 18.6781 12.391 18.5344 12.5346C18.3906 12.6782 18.2125 12.75 18 12.75H5.99997Z"
      fill="currentColor"
    />
  </svg>
</div>
`;
